staff/almu-itur/app-template/news.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>News</title>
+</head>
+<body>
+
+    </header>
+        <nav>
+            <ul>
+                <li><a href=".\home.html">Home</a></li>
+                <li><a href=".\profile.html">Profile</a></li>
+                <li><a href=".\about.html">About</a></li>
+                <li><a href=".\news.html">News</a></li>
+                <li><a href=".\landing.html">Landing</a></li>
+                <li><a href=".\login.html">Log in</a></li>
+                <li><a href=".\register.html">Register</a></li>
+                <li><a href=".\contact.html">Contact</a></li>
+            </ul>
+        </nav>
+    </header>
+
+    <h1>NEWS</h1>
+
+    <main>
+        <article>
+            <header>
+                <h1>Main heading</h1>
+                <h3>Secondary heading</h3>
+                <time datetime='2018-1-1'>January 1st</time>
+            </header>
+            <picture srcset="image.jpg"></picture>
+            <p>Lorem Ipsum dolor set amet....</p>
+            <figure>
+                <img src="image-article.jpg">
```

Comment:
The alt attribute is required.

[alt information](https://www.w3schools.com/TAGS/att_img_alt.asp)

Reply:
Ok